bool sky_light
chunk_meta_msg[] metadata
uint8[] data
uint64 ROStimestamp
int64 MCtimestamp
